grammars/javascript.cson

```diff
@@ -633,6 +833,14 @@
   }
 ]
 'repository':
+  'operators':
```

Comment:
If we're going to make this a repository it should probably go beneath `numbers`